codecov_cli/helpers/request.py

```diff
@@ -1,24 +1,46 @@
 import logging
 import uuid
+from time import sleep
 
 import requests
 
 from codecov_cli.types import RequestError, RequestResult
 
 logger = logging.getLogger("codecovcli")
 
+MAX_RETRIES = 3
 
+
+def backoff_time(curr_retry):
+    return 2 ** (curr_retry - 1)
```

Comment:
This will return 1, 2, and 4 seconds for the 3 retries.  Is that a helpful backoff or should it grow faster (i.e. `5 ** n` or something)?  Alternatively we could just sleep for a fixed 3 seconds or something every time.

Reply:
Technically it returns 0.5, 1 and 2 seconds (because the first value is 0).
I wasn't sure what to put here either, honestly... But I think it's fine, specially with timeout errors, because there's the time of the request as well.
We can also increase the number of retries to 5

Reply:
Being an iteractive tool I don't think we should use too big of a backoff period